integration-tests/README.md

```diff
@@ -68,16 +68,48 @@ can either be 8 or 11.
 Druid's configuration (using Docker) can be overrided by providing -Doverride.config.path=<PATH_TO_FILE>. 
 The file must contain one property per line, the key must start with `druid_` and the format should be snake case. 
 
+## Running tests by docker-compose + mvn
+
+Build druid-cluster, druid-hadoop docker images. From root module run maven command:
```

Comment:
Thanks for adding "Build druid-cluster, druid-hadoop docker images. From root module run maven command:" and "Run druid cluster by docker-compose:". I think it might be nice if we document the variation of yml we have here too

Reply:
Added details about all yamls